sccm-currentbranch/azuredeploy.json

```diff
@@ -89,10 +89,28 @@
       "Premium_LRS"
     ],
     "vmSize": [
-      "Standard_B2ms",
-      "Standard_B2ms",
-      "Standard_B2ms",
+      "Standard_B2s",
+      "Standard_B2s",
+      "Standard_B2s",
       "Standard_B2ms"
+    ],
+    "vmpublisher": [
```

Comment:
let's put all the image reference properties into the same object variable...  see: https://github.com/Azure/azure-quickstart-templates/blob/master/201-vm-msi/nestedtemplates/createVM.json#L118-L137

Reply:
Thanks for the suggestion , I've fixed this issue